phlib/hndlinfo.c

```diff
@@ -836,95 +836,113 @@ _Callback_ PPH_STRING PhStdGetClientIdName(
     _In_ PCLIENT_ID ClientId
     )
 {
-    static PH_QUEUED_LOCK cachedProcessesLock = PH_QUEUED_LOCK_INIT;
-    static PVOID processes = NULL;
-    static ULONG64 lastProcessesTickCount = 0;
-
-    PPH_STRING name;
-    ULONG64 tickCount;
-    PSYSTEM_PROCESS_INFORMATION processInfo;
+    return PhStdGetClientIdNameEx(ClientId, NULL);
+}
 
-    // Get a new process list only if 2 seconds have passed since the last update.
-    tickCount = NtGetTickCount64();
+PPH_STRING PhStdGetClientIdNameEx(
+    _In_ PCLIENT_ID ClientId,
+    _In_opt_ PPH_STRING ProcessName
+    )
+{
+    PPH_STRING result;
+    PPH_STRING processName = NULL;
+    PPH_STRING threadName = NULL;
+    PH_STRINGREF processNameRef;
+    PH_STRINGREF threadNameRef;
+    HANDLE hProcess, hThread;
```

Comment:
Don't use hProcess, hThread on the same line. It should use processHandle, threadHandle.